e2e/paths/caseSubmitted_test.js

```diff
@@ -18,3 +19,9 @@ Scenario('Submitting case', (I, caseViewPage) => {
   I.reloadPage();
   I.see('Barnet_Council_v_Smith.pdf');
 });
+
+Scenario('Cannot submit a case unless consent is given', I => {
+  I.see('I, Tester Tester, believe that the facts stated in this application are true.');  
```

Comment:
Remember to update after Mark's change is merged.

Reply:
That still needs to be replaced.

Reply:
e2e tests have been updated